types/test/model.ts

```diff
@@ -10,3 +11,6 @@ class MyModel extends Model {
 }
 
 const assoc: Association = MyModel.associations.other;
+
+const Instance: MyModel = new MyModel({ int: 10 });
+const int: IntegerDataType = Instance.get('int');
```

Comment:
`IntegerDataType` is not used for model properties but only inside of sequelize, you should use something like `number`.

Reply:
Should be good now.